src/assets/styles/base.scss

```diff
@@ -249,8 +249,10 @@ ul.list {
 .invalid-section {
   border-left: 3px solid $app-red !important;
 
-  .title-label {
-    color: $app-red !important;
```

Comment:
Has this styling been replaced locally at the component level?
I know this class is relied on for the labels being red when the parent section is invalid.

Reply:
Like this?

![image](https://user-images.githubusercontent.com/10002889/141222658-88560141-245e-4318-980d-4e06c8b75941.png)

It still works!

Reply:
See also `CustodianOfRecords` and `DissolutionStatements`, the only other instances i can find of the class.

Reply:
Can't test atm but you're right: it is used by a few components. So I restored this.